### DISCUSSION TITLE
Primefaces default Keyboard issue

### DISCUSSION THREAD

**Discussion by pravinrane38** [primefaces/community#1411] [PrimeFaces] [answered]
Primefaces default Keyboard issue :
Keyboard is not showing properly, its display weird on click
<img width="934" alt="keyboard" src="https://github.com/primefaces/community/assets/61691884/eaee672e-7665-4b3d-8518-02e7ebf7acf1">

**Comment by melloware** [comment]
It looks OK to me?  

![image](https://github.com/primefaces/community/assets/4399574/258a6b24-4955-4a00-a1b6-5c8d1bb09567)


However the keyboard does not appear to be responsive so for it me it is only messed up in mobile size browsers?

![image](https://github.com/primefaces/community/assets/4399574/5efa02c4-995f-4568-8ca7-cf841bb0caf7)

**Reply by Rapster** [reply, marked as answer]
Maybe time for an upgrade, the plugin is 6 years old. What about https://hodgef.com/simple-keyboard/demos/ ?

**Reply by melloware** [reply]
Wow that looks nice!

**Reply by Rapster** [reply]
I saw this one also: https://furcan.github.io/KioskBoard/ but not been updated for a year

**Reply by melloware** [reply]
I opened this ticket if you want to follow there: https://github.com/primefaces/primefaces/issues/11669
